README.md

```diff
@@ -309,31 +325,31 @@ var companies []*models.CompanyModel
 // Campaign object is optional, but useful if you want to track ROI of acquisition channels
 // See https://www.moesif.com/docs/api#update-a-company for campaign schema
 campaign := models.CampaignModel {
-  UtmSource: "google",
-  UtmMedium: "cpc", 
-  UtmCampaign: "adwords",
-  UtmTerm: "api+tooling",
-  UtmContent: "landing",
+  UtmSource: literalFieldValue("google"),
+  UtmMedium: literalFieldValue("cpc"), 
+  UtmCampaign: literalFieldValue("adwords"),
+  UtmTerm: literalFieldValue("api+tooling"),
+  UtmContent: literalFieldValue("landing"),
 }
   
 // metadata can be any custom dictionary
 metadata := map[string]interface{}{
-  "org_name", "Acme, Inc",
-  "plan_name", "Free",
-  "deal_stage", "Lead",
-  "mrr", 24000,
-  "demographics", map[string]interface{}{
-      "alexa_ranking", 500000,
-      "employee_count", 47,
+  "org_name": "Acme, Inc",
+  "plan_name": "Free",
+  "deal_stage": "Lead",
+  "mrr": 24000,
+  "demographics": map[string]interface{}{
+      "alexa_ranking": 500000,
+      "employee_count": 47,
   },
 }
 
 // Prepare company model
 companyA := models.CompanyModel{
 	CompanyId:		  "67890",	// The only required field is your company id
```

Comment:
do you need literalFieldValue("67890"), here? whats the reason behind changing to use address for constants

Reply:
Nope, it is not required. CompanyId is the required field in the CompanyModel while CompanyDomain is an optional field (type: *string). So we can't assign string value to CompanyDomain field, so we use literalFieldValue helper function to make it an optional value.

Reply:
we changed companyDomain from required to optional now? or earlier it was missed?

Reply:
It was optional from the beginning but the readme was missing it.

Reply:
ok